src/Snppts/Snippets/MauiMsDosUI.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Snppts.Authors;
+using Snppts.Infrastructure;
+using System.Collections.Generic;
+using System;
+using Snppts.Extensions;
+
+namespace Snppts.Snippets
+{
+    public class MauiMsDosUI : IAmASnippet
+    {
+        public string Slug => "maui-ms-dos";
+        public string Title => "Maui MS Dos";
+        public string GithubRepoName => "jm-parent/MSDos4Ever";
+        public bool ContainsAndroidSample => true;
+        public bool ContainsiOSSample => true;
+        public bool ContainsUWPSample => true;
+
+        public string Description => "For all developers who want to make an application with a modern and clean design.";
+
+        public IAmAnAuthor AuthorInfo => new JeanMarieParent();
+
+        public IEnumerable<Uri> ImageUris => new List<Uri>
+        {
+            //Mobile
+            new Uri("https://github.com/jm-parent/MSDos4Ever/blob/master/MSDos4Ever/SnpptsImages/msdos_ios_landscape.png?raw=true"),
+            new Uri("https://github.com/jm-parent/MSDos4Ever/blob/master/MSDos4Ever/SnpptsImages/msdos_ios_portrait.png?raw=true"),
+            //Windows
+            new Uri("https://github.com/jm-parent/MSDos4Ever/blob/master/MSDos4Ever/SnpptsImages/msdos_windows.png?raw=true")
```

Comment:
Please make them all portrait, our very advances web application will probably break layout if we stick landscape stuff in it 😅

Reply:
Oh ok thanks! Perfect!